File: tech-start-test-angular/src/app/model/Manufacturer.ts
import { deserialize } from "cerialize";

export class Manufacturer{
    @deserialize id: string;
    @deserialize name: string;
    @deserialize address: string;
    @deserialize contact: string;
  
    constructor(id: string, name : string, address : string, contact: string) {
        this.id = id;
        this.name = name;
        this.address = address;
        this.contact = contact;
      }
}

File: tech-start-test-api/src/main/java/com/test/techstarttestapi/model/Manufacturer.java
package com.test.techstarttestapi.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;
import org.springframework.data.annotation.Id;

@NoArgsConstructor
@AllArgsConstructor
@Data
@ToString
public class Manufacturer {
    @Id
    private String Id;
    private String name;
    private String Address;
    private String Contact;
}
